config/config.go

```diff
@@ -867,11 +867,25 @@ func PrintConfig() error {
 	if err != nil {
 		return err
 	}
-	fmt.Fprintln(os.Stderr,
-		"================ Pelican Configuration ================\n",
-		string(bytes),
-		"\n",
-		"============= End of Pelican Configuration ============")
+
+	originalFormatter := log.StandardLogger().Formatter
+
```

Comment:
I think it'd be slightly more clear that you intend to re-set this value later if you did a `defer func(){log.SetFormatter(originalFormatter)}()`. This also protects a future where someone might add an error-producing function call between here and the reset without realizing they need to handle resetting the log formatter differently.